src/status_im/contexts/wallet/common/utils/wallet_connect.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im.contexts.wallet.common.utils.wallet-connect
+  (:require ["@walletconnect/core" :refer [Core]]
+            ["@walletconnect/web3wallet" :refer [Web3Wallet]]
+            [promesa.core :as promesa]
+            [status-im.config :as config]))
+
+(defn- wallet-connect-core
+  []
+  (Core. (clj->js {:project-id config/wallet-connect-project-id})))
```

Comment:
Are we sure the option is in kebab-case? Here https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/core/src/core.ts#L67 I see there it should be `projectId`.

Reply:
Updated